blog/poc3.md

```diff
@@ -0,0 +1,127 @@
+Not only can pss be used for slow-scale chat messaging, but also for secure file transfer, notifications and collaborative protocols like [CRASH proof of custody]() used in storage incentivisation.
+
+PSS is fully featured yet experimental on the new POC3 network. We are collaborating closely with companies and projects that want to use pss to build mid-layer infrastructure. [Mainframe](http://mainframe.com) is building a slack-alternative collaborative group
+ communications tool ([Onyx](https://blog.mainframe.com/mainframe-pre-alpha-release-fca532317111)) and their web3 SDK, and [Status](http://status.im) have expressed interest in building it into their mobile chat.
+
+## Mutability
+
+Using hashes as content addresses allows us to view hashes as addresses at which content is expected to be found.
+Since hashes can be assumed to be collision free, they are bound to one specific version of a content. Hash addressing is therefore immutable in the strong sense that you cannot even express mutable content: "changing the content changes the hash".
+
+Users of the web, however, are accustomed to mutable resources, looking up domains and expect to see the most up to date version of the 'site'. Mutable resources are made possible by the ethereum name service (ENS) and Mutable Resource Updates (MRU).
+The ENS is a smart contract on the ethereum blockchain which enables domain owners to register a content reference to their domain.
+Using ENS for domain name resolution, the url scheme provides
+content retrieval based on mnemonic (or branded) names, much like the DNS of the world wide web, but without servers.
+Mutable Resource Update is, on the other hand, an off-chain solution for communicating updates to a resource, it offers cheaper and potentially faster updates than ENS, yet the updates can be consolidated on ENS by any third party willing to pay for the transaction.
+
+Here is how it works: a publisher creates a mutable resource at time `t` for the name "my daily tweets", that she intends to update every day once (note this in itself not a promise). By hashing together `ID := SHA3(publisher_pubkey, name, start, period)`, we obtain a so called metadata chunk.
+A publisher creates new version of a mutable resource by serialising `(name, start, period, version)` and signing it.
+This simple system is surprisingly powerful and is envisioned to be at the core of implementing group chat, shared FUSE state, and [poor man's consensus](3rd orange paper) (soft channel deposit).
+
+MRU is an experimental feature in current POC3 testnet and is still  undergoing changes.
+
+## Drop box
+
+We mentioned that thanks to manifests, directory trees and collections represented by manifests can be mapped uniquely. This does not only allow upload and download into directory but also use FUSE integration to mount a swarm manifest as a local file system.
```

Comment:
I suggest ...directory trees and collections (represented by manifests) can be mapped uniquely (brackets for better readabilty)